Content.Server/GameObjects/Components/Atmos/AtmoExposableComponent.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Content.Server.Atmos;
+using Content.Server.GameObjects.Components.Temperature;
+using Robust.Shared.GameObjects;
+
+namespace Content.Server.GameObjects.Components.Atmos
+{
+    /// <summary>
+    /// Represents that entity can be exposed to Atmo
+    /// </summary>
+    [RegisterComponent]
+    public class AtmoExposableComponent
```

Comment:
This name isn't great, reading `Atmo` instead of `Atmos` feels really weird, and it's not clear what this does at a first glance. Perhaps a name like `AtmosTemperatureExposable` would be best.